shell/client/grain-client.js

```diff
@@ -438,6 +440,58 @@ Template.shareableLinkTab.events({
   },
 });
 
+Template.grainShareButton.onRendered(() => {
+  if (!Meteor._localStorage.getItem("userNeedsShareAccessHint")) {
+    return;
+  }
+
+  // Don't show hint on $mobile because the top bar isn't really a top bar in that case.
+  if (window.innerWidth <= 900) {
```

Comment:
But in general, I agree that's a risk.

On the upside, if you `git grep` for `$mobile` you'll find this.

To brainstorm options:

- We could ask some behavioral question of the DOM that'll only ever be true in "desktop" mode, like if some element is visible.
- We could, in one standard place, set a `className` on the `body` element via JS, to label the body as being in `mobile` mode, and then depend on that CSS class having been applied to the body.
- We could, in the SCSS files next to the definition of `$mobile`, mention that anyone modifying the breakpoint should `git grep -F "$mobile"` to make sure that they understand other places that the value was hard-coded.

The last of these is my favorite option. I'm open to a variety of options, though. Curious for your input @kentonv .